Src/BootCamp.Chapter/Logger/ConsoleLogger.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace BootCamp.Chapter.Logger
+{
+    class ConsoleLogger : ILogger
+    {
+        public void Log(string msg)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'Log' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5147695)